README.md

```diff
@@ -0,0 +1,471 @@
+# API Documentation
+
+## Introduction
+
+This API allows users find information about people in the neighbourhood.
+Users can:
+
+- store people, houses and addresses
+- look up a house, its address and owner
+- look up people in our neighbourhood within certain age brackets and with specific household sizes
+
+## Database schema
+
```

Comment:
What a fantastic read me! I really enjoyed how you broke down the exercise requests into components e.g., store people, store addresses etc. You offered an example route, method and pathway. You even offered an sample call. Great job you two!